Cubical/Categories/Displayed/HLevels.agda

```diff
@@ -0,0 +1,75 @@
+{-# OPTIONS --safe #-}
```

Comment:
Maybe a comment that this file is about ```HLevels``` of ```Hom```s would be good here.

Reply:
Added